parlai/core/pytorch_data_teacher.py

```diff
@@ -586,6 +587,11 @@ def __init__(self, opt, shared=None):
                         ('stream' in self.datatype and not opt.get('shuffle')))
         if self.ordered:
             # force index for ordered, so that we see every example
+            warn_once(
+                '\nNote: You are using PytorchDataTeacher with ordered '
```

Comment:
I think the extra newline is causing the problem?

Reply:
sadly, it does not